configs/8.3/packages.lst

```diff
@@ -108,6 +107,7 @@ psmisc
 pth
 pygpgme
 python
+python2-newt
```

Comment:
It is indeed not necessary to have it listed explicitly, it is pulled by `system-config-firewall-tui`.

Reply:
Done